README.rst

```diff
@@ -15,30 +15,43 @@ learning, automatic differentiation, and optimization of hybrid quantum-classica
 Installation
 ============
 
+We suggest first installing Kokkos with the wanted configuration following the instruction found in the `Kokkos documentation <https://kokkos.github.io/kokkos-core-wiki/building.html>`_.
+Next, append the install location to ``CMAKE_PREFIX_PATH``.
+If an installation is not found, our builder will install it from scratch nevertheless.
+
+The simplest way to install PennyLane-Lightning-Kokkos (OpenMP backend) is using ``pip``.
+
 .. code-block:: console
 
-   cmake -Bbuild -DKokkos_ENABLE_OPENMP=On -DPLKOKKOS_BUILD_TESTS=On -G Ninja
-   cmake --build ./build
+   CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" python -m pip install .
 
-You can install the python interface with:
+or for an editable ``pip`` installation with:
 
 .. code-block:: console
 
-   python setup.py build_ext --backend="OPENMP"
+   CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" python -m pip install -e .
+
+Alternatively, you can install the python interface with:
+
+.. code-block:: console
+
+   CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" python setup.py build_ext
    python setup.py bdist_wheel
    pip install ./dist/PennyLane*.whl --force-reinstall
 
-
-or for an editable `pip` installation with:
+To build the plugin directly with CMake:
 
 .. code-block:: console
 
-   BACKEND="OPENMP" python -m pip install -e .
+   cmake -B build -DKokkos_ENABLE_OPENMP=ON -DPLKOKKOS_BUILD_TESTS=ON -G Ninja
+   cmake --build build
 
-
-Supported backend options are "SERIAL", "OPENMP", "THREADS", "HIP" and "CUDA". For "HIP" and "CUDA", the appropriate software stacks are required to enable compilation and subsequent use.
-For explicit targeting of a given supported architecture, the environment variable `ARCH` can also be specified which directly sets the `-DKokkos_ARCH_{...}=ON` build option. Note that "THREADS"
-backend is not recommended since `Kokkos <https://github.com/kokkos/kokkos-core-wiki/blob/17f08a6483937c26e14ec3c93a2aa40e4ce081ce/docs/source/ProgrammingGuide/Initialization.md?plain=1#L67>`_ does not guarantee its safety.
+Supported backend options are "SERIAL", "OPENMP", "THREADS", "HIP" and "CUDA" and the corresponding build switches are ``-DKokkos_ENABLE_BACKEND=ON``, where one needs to replace ``BACKEND``.
+One can activate simultaneously one serial, one parallel host and one parallel host backend, but not two of any category at the same time.
```

Comment:
This sentence is kind of confusing. Could you please re-write that?

Reply:
First sentence, second, or both?

Reply:
How about 
```
Several backends can be simultaneously available, but there are some restrictions.
One has to choose between "OPENMP" and "THREADS".
One has to choose between "HIP" and "CUDA".
"SERIAL" can always be activated.
```
?

Reply:
Ah, I just realized, I meant to write
```
One can activate simultaneously one serial, one parallel host and one parallel device backend, but not two of any category at the same time.
```
What do you prefer?

Reply:
The last one is nice. You could maybe "merge" the two options in:
`One can activate simultaneously one serial, one parallel host (like threads or openMP) and one parallel device backend (like HIP or CUDA), but not two of any category at the same time.`
or
`One can activate simultaneously one serial, one parallel CPU host (like threads or openMP) and one parallel GPU device backend (like HIP or CUDA), but not two of any category at the same time.`

Reply:
Indeed, that is the best. I'll introduce your changes.